### PR TITLE
Us withholding tax guide

### PR DESCRIPTION
**Pull request by chrislwk**
# Checklist

- [X] I have spellchecked my changes.
- [X] I have updated the API Documentation CHANGELOG where applicable.
- [X] I have added my guides to Welcome.md where applicable
- [X] I have added my guides to toc.json where applicable.

### CODE REVIEW

File: docs/US Withholding Tax/Basics.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+To ensure the customer pays the correct amount of tax, they can complete a W-8 BEN form. 
+
+The W-8 BEN form is a declaration by the customer and includes information such as their personal details and tax residency. The form remains active until the end of the third calendar year after the form is signed (i.e. if the form is signed in September 2022, it is valid until 31st December 2025). At this point a customer will need to complete a new form to benefit from reduced rates of withholding tax.
+
+WealthKernel provides a service for Tenants which will generate all of the required information to complete the W-8 BEN form to be presented to a customer. If all of the information presented to a customer is correct, the customer can attest to the information. Tenant's can submit a customer's attestation which will create a PDF copy of the W-8 BEN form with a digital signature of the customer. The customer can access a copy of the W-8 BEN form should they require for their records.
```
Comment (LukeBillo) [inline docs/US Withholding Tax/Basics.md:9]:
```suggestion
WealthKernel provides a service for Tenants which will generate all of the required information to complete the W-8 BEN form to be presented to a customer. If all of the information presented to a customer is correct, the customer can attest to the information. Tenants can submit a customer's attestation which will create a PDF copy of the W-8 BEN form with a digital signature of the customer. The customer can access a copy of the W-8 BEN form should they require for their records. 
```

File: docs/US Withholding Tax/Basics.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+The W-8 BEN form is a declaration by the customer and includes information such as their personal details and tax residency. The form remains active until the end of the third calendar year after the form is signed (i.e. if the form is signed in September 2022, it is valid until 31st December 2025). At this point a customer will need to complete a new form to benefit from reduced rates of withholding tax.
+
+WealthKernel provides a service for Tenants which will generate all of the required information to complete the W-8 BEN form to be presented to a customer. If all of the information presented to a customer is correct, the customer can attest to the information. Tenant's can submit a customer's attestation which will create a PDF copy of the W-8 BEN form with a digital signature of the customer. The customer can access a copy of the W-8 BEN form should they require for their records. 
+
+```mermaid
+sequenceDiagram
+    participant C as Customer
+    participant T as Tenant
+    participant W as WealthKernel
+    C->>T: Selects a US Share or model containing a US Share to buy
```
Comment (LukeBillo) [inline docs/US Withholding Tax/Basics.md:16]: This is suggesting that immediately after they select a US share / model they'd send an order which drafts the W-8BEN - this is not the workflow we've got.

I'd make it a bit clearer, something like this, and suggest the flow for their application:
```mermaid
sequenceDiagram
    participant P as Party
    participant T as Tenant
    participant W as WealthKernel
    P->>T: Selects a US Share or model containing a US Share to buy
    T->>P: Inform party about benefitting from a W-8BEN
    P->>T: Requests draft W-8BEN
    T->>W: Requests draft W-8BEN for party
    W->>T: Provides draft with party data required for attestation
    T->>P: Presents information to party
    P->>T: Attests to information
    T->>W: Confirmation of party attestation
    W->>W: Log information 
    W->>W: Generate and store PDF
opt   
    T->>W: Request W-8BEN PDF
    W->>T: PDF returned
    T->>P: Display W-8BEN PDF document to party
end
```

Note that I've also renamed customer to "party" as this is the terminology we'd use in our API documentation

File: docs/Welcome.md [hunk 3]
```diff
@@ -56,6 +61,10 @@
 
 1. [Basics](./transactions/Basics.md)
 
+## US Withholding Tax
+
+1. [Basics](./us-withholding-tax/Basics.md)
```
Comment (LukeBillo) [inline docs/Welcome.md:66]: Does this also need a FAQ? I imagine tenants might ask questions along the lines of:

1. What if the W-8BEN draft has incorrect information?
2. What happens when the W-8BEN expires?
3. Once the party has attested the W-8BEN has the correct information, when does the W-8BEN become valid?
4. What countries do you support for W-8BENs right now?

File: docs/us-withholding-tax/Basics.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+To ensure the customer pays the correct amount of tax, they can complete a W-8 BEN form. 
+
+The W-8 BEN form is a declaration by the customer and includes information such as their personal details and tax residency. The form remains active until the end of the third calendar year after the form is signed (i.e. if the form is signed in September 2022, it is valid until 31st December 2025). At this point a customer will need to complete a new form to benefit from reduced rates of withholding tax.
+
+WealthKernel provides a service for Tenants which will generate all of the required information to complete the W-8 BEN form to be presented to a customer. If all of the information presented to a customer is correct, the customer can attest to the information. Tenant's can submit a customer's attestation which will create a PDF copy of the W-8 BEN form with a digital signature of the customer. The customer can access a copy of the W-8 BEN form should they require for their records. 
+
+```mermaid
+sequenceDiagram
```
Comment (LukeBillo) [inline docs/us-withholding-tax/Basics.md:16]: This is suggesting that immediately after they select a US share / model they'd send an order which drafts the W-8BEN - this is not the workflow we've got.

I'd make it a bit clearer, something like this, and suggest the flow for their application:
```mermaid
sequenceDiagram
    participant P as Party
    participant T as Tenant
    participant W as WealthKernel
    P->>T: Selects a US Share or model containing a US Share to buy
    T->>P: Inform party about benefitting from a W-8BEN
    P->>T: Requests draft W-8BEN
    T->>W: Requests draft W-8BEN for party
    W->>T: Provides draft with party data required for attestation
    T->>P: Presents information to party
    P->>T: Attests to information
    T->>W: Confirmation of party attestation
    W->>W: Log information 
    W->>W: Generate and store PDF
opt   
    T->>W: Request W-8BEN PDF
    W->>T: PDF returned
    T->>P: Display W-8BEN PDF document to party
end
```

Note that I've also renamed customer to "party" as this is the terminology we'd use in our API documentation

File: docs/us-withholding-tax/Basics.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+To ensure the customer pays the correct amount of tax, they can complete a W-8 BEN form. 
+
+The W-8 BEN form is a declaration by the customer and includes information such as their personal details and tax residency. The form remains active until the end of the third calendar year after the form is signed (i.e. if the form is signed in September 2022, it is valid until 31st December 2025). At this point a customer will need to complete a new form to benefit from reduced rates of withholding tax.
+
+WealthKernel provides a service for Tenants which will generate all of the required information to complete the W-8 BEN form to be presented to a customer. If all of the information presented to a customer is correct, the customer can attest to the information. Tenant's can submit a customer's attestation which will create a PDF copy of the W-8 BEN form with a digital signature of the customer. The customer can access a copy of the W-8 BEN form should they require for their records.
```
Comment (LukeBillo) [inline docs/us-withholding-tax/Basics.md:13]:
```suggestion
WealthKernel provides a service for Tenants which will generate all of the required information to complete the W-8 BEN form to be presented to a customer. If all of the information presented to a customer is correct, the customer can attest to the information. Tenants can submit a customer's attestation which will create a PDF copy of the W-8 BEN form with a digital signature of the customer. The customer can access a copy of the W-8 BEN form should they require for their records. 
```

File: docs/Welcome.md (hunk 3 above)
Comment (chrislwk) [reply docs/Welcome.md:66]: FAQs addd

**Comment by LukeBillo** [comment]
Closing as its a duplicate of #65

**Pull request closed without merging**